### PR TITLE
add file caching

### PR DESCRIPTION
**Pull request by restrry**
https://github.com/mrsum/webpack-svgstore-plugin/issues/54

### CODE REVIEW

File: helpers/utils.js [hunk 1]
```diff
@@ -253,6 +255,28 @@ var _parseFiles = function(files, options) {
...
+var _filesChanged = function(files){
+  var result = false;
+  try{
+    for (var i = 0; i < files.length; i++) {
+      var filepath = files[i];
+        var fstat = fs.statSync(filepath);
+        if(fstat.mtime > (fileCache[filepath] || 0)){
+          fileCache[filepath] = fstat.mtime;
+          result = true;
+        }
```
Comment (lgordey) [inline helpers/utils.js]: extra tab from 267 to 271

**Comment by lgordey** [comment]
@restrry Can you add some tests for this please? :)

**Comment by restrry** [comment]
added

**Comment by restrry** [comment]
ping

**Pull request merged**